### ISSUE TITLE
TwitterDigits not defined

### ISSUE THREAD

**Issue by Gasorimplat** [nifino/twitter-digits-plugin#2]
Hello, i'm testing integration of Digits with one test app in ionic. After follow all steps, when run the app in the console appear 'ReferenceError: TwitterDigits is not defined', how i add the reference to TwitterDigits?
(Sorry!, im new on this)

**Comment by osricwong** [comment]
yes, i also want to start in ionic from integration of digits, this message is also prompted in console.
